contributing-guides/style-guide.md

```diff
@@ -330,7 +330,16 @@ For instance, instead of `Listing all files:`, `List all files:` can be used as
  `ls`
 ```
 
-## Emphasis
+## Not rendered syntax
+
+Any syntax not documented in this guide or the client specification will **not** be rendered in most of our clients. See common cases below.
+
+### Links and images
+
+- Do not use Markdown links (\[link text\](link url)) or images in the pages, they are not part of the tldr client specification and thus not supported by most of our clients.
+- If you want to use a link in a page, enclose it within angled brackets (e.g. <https://example.org>).
```

Comment:
Right but for the wrong reasons? Also, how about this rewording? Too many `do not`s on this page can give the wrong impression when reading.
```suggestion
Use angled brackets (e.g. <https://example.org>) when including links in pages.
```
I would also suggest that just because something isn't supported by clients isn't a good enough reason on its own to avoid it here.

Do we really need to say that images are not allowed? The linter won't even accept them, and unless I'm wrong we haven't ever had a PR that adds an image to a tldr page....

Reply:
> Do we really need to say that images are not allowed? The linter won't even accept them, and unless I'm wrong we haven't ever had a PR that adds an image to a tldr page....

Afaik we haven't had any PR having images using the Markdown rendering format `![alt text](link)`. And I don't think tldr-lint has a rule for it.